components/brave_wallet/browser/bitcoin/bitcoin_tx_manager.cc

```diff
@@ -56,6 +56,7 @@ void BitcoinTxManager::AddUnapprovedTransaction(
     const absl::optional<url::Origin>& origin,
     AddUnapprovedTransactionCallback callback) {
   const auto& btc_tx_data = tx_data_union->get_btc_tx_data();
+  // TODO(nvonpentz) Ensure to address is not an OFAC banned address.
```

Comment:
This should not be done in each AddUnapprovedTransaction in *TxManager, this should be done in `TxService::AddUnapprovedTransactionWithOrigin` instead.